nox/virtualenv.py

```diff
@@ -182,6 +196,92 @@ def _resolved_interpreter(self):
         self._resolved = InterpreterNotFound(self.interpreter)
         raise self._resolved
 
+    @property
+    def _runtime_interpreter(self):
+        """
+        Returns the interpreter to use when shelling out, ``sys.executable``,
+        unless Nox was frozen with PyInstaller. In the later case, returns
+        system python interpreter matching the interpreter used when Nox was
+        frozen with PyInstaller or as low as Python 2.7, if found."""
```

Comment:
Instead

```python
"""The Python interpreter to invoke from the shell.

This is typically ``sys.executable``, but there are a few exceptions.
If Nox was "frozen" with PyInstaller, then a special path will be used.
If no executable can be resolved, this will raise a stored exception.
"""
```

Reply:
ok